packages/cozy-client/src/store/documents.js

```diff
@@ -99,7 +99,14 @@ const documents = (state = {}, action) => {
 
 export default documents
 
-// selector
+/**
+ * Get documents from state, by their ids
```

Comment:
```suggestion
 * Get document from state, by its id
```

Reply:
Uh, I have some copy paste lazyness today... Thanks for your attention